ue5osc/__init__.py

```diff
@@ -1,21 +1,103 @@
 from pythonosc import udp_client
+from time import sleep
+
+from ue5osc.osc_dispatcher import DispatchHandler
+from pythonosc.osc_server import BlockingOSCUDPServer
+from PIL import Image
+
+import matplotlib.pyplot as plt
 
 
 class OSCSender:
-    def __init__(self, ip: str, port: int):
+    """Handles interaction between the UE5 environment and the a program."""
+
+    # Initializing the OSC communicator
+    def __init__(self, ip: str, port: int, directory: str):
+        self.path = directory

```

Comment:
Use proper docstrings.

```suggestion
    def __init__(self, ip: str, port: int, directory: str):
        "Initialize OSC server."
        self.path = directory

```